dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -53,6 +53,9 @@ static struct Context {
 
 static const char *LOG_TAG = "DatadogNdkCrashReporter";
 static pthread_mutex_t handler_mutex = PTHREAD_MUTEX_INITIALIZER;
+static const uint8_t tracking_consent_pending = 0;
+static const uint8_t tracking_consent_granted = 1;
+static uint8_t tracking_consent = tracking_consent_pending; // 0 - PENDING, 1 - GRANTED, 2 - NOT-GRANTED
```

Comment:
Shouldn't you add a 
```cpp
static const uint8_t tracking_consent_not_granted = 2;
```

Reply:
it is not used, added in the first place but realised we are not using that value at all.